### PR TITLE
Development

### PR DESCRIPTION
**Pull request by csieffert**
Removed all gview changes again and made sure the vcf2.. sub module was updated.  Let me know if it was all done correctly.  Thanks!

### CODE REVIEW

**Comment by apetkau** [comment]
Awesome, thanks.  Just note, before you update vcf2pseudoalignment can you make sure to keep that branch in sync with my development branch at https://github.com/apetkau/vcf2pseudoalignment/commits/development.  That is, update to commit https://github.com/apetkau/vcf2pseudoalignment/commit/b2bc84424ba1fca14cb96354cf698fa869eae71a

**Comment by csieffert** [comment]
Hi aaron, I have synced with your development branch.  Would you like me to create a new pull request?

**Comment by csieffert** [comment]
Thanks for noticing that aaron, everything should be synced now.

**Comment by apetkau** [comment]
Awesome, this looks great.  Thanks.  The tests all pass for me.

I did notice there are a few other areas where `gview` is mentioned though, mainly in the documentation (for example in the INSTALL.md).  Could you remove those references as well?

You can check for all the areas to remove by running:

```
$ grep -R 'gview' *
doc/install-ubuntu/ubuntu-install-instructions.sh:# gview
doc/install-ubuntu/ubuntu-install-instructions.sh:wget https://www.gview.ca/files/release/1.7/gview-1.7.zip
doc/install-ubuntu/ubuntu-install-instructions.sh:unzip gview-1.7.zip
doc/install-ubuntu/ubuntu-install-instructions.sh:chmod +x /opt/gview/gview.jar
doc/install-ubuntu/ubuntu-install-instructions.sh:export PATH=/opt/gview/:$PATH
INSTALL.md:* [GView](https://www.gview.ca)
INSTALL.md:             gview: /path/to/gview.jar
INSTALL.md:     gview_style: '/path/to/core-phylogenomics/etc/original.gss'
scripts/snp_phylogenomics_control.pl:=item B<vcf2core>:  Generates gview file(s)
```

Beyond that, everything looks great.  Thanks :)

**Comment by csieffert** [comment]
I've removed the gview references from the install instructions and the grep command is not showing any additional references.  Thanks!

File: scripts/snp_phylogenomics_control.pl [hunk 1]
```diff
@@ -717,8 +717,6 @@ =head2 MAPPING
 
 =item B<pseudoalign>:  Generates pseudoalignment file.
 
-=item B<vcf2core>:  Generates gview file(s)
```
Comment (apetkau) [inline scripts/snp_phylogenomics_control.pl]: Could you actually re-add this comment, but change to **Computes percent of genome used for analysis.**?

**Comment by csieffert** [comment]
Comment is back in

**Comment by apetkau** [comment]
This looks great and everything passes.  Merging the code in.  Thanks.

**Comment by csieffert** [comment]
Great, thanks

**Pull request merged**